core/src/autogluon/core/hpo/executors.py

```diff
@@ -273,7 +273,16 @@ def get_hpo_results(self, model_name, model_path_root, **kwargs):
             file_id = trial_id  # unique identifier to files from this trial
             trial_model_name = model_name + os.path.sep + file_id
             trial_model_path = model_path_root + trial_model_name + os.path.sep
-            hpo_models[trial_model_name] = trial_model_path
+            hpo_models[trial_model_name] = dict(
+                path=trial_model_path
+            )
+
+            hpo_models[trial_model_name] = dict(
+                path=trial_model_path,
+                val_score=validation_performance,
+                trial=trial,
+                # hyperparameters=???  # TODO: how to get hyperparameters of a trial with ray?
```

Comment:
Do we need tracking issue for this?

Reply:
@yinweisu if you can provide the code to get hyperparameters here then we don't need to revisit this.